<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="E:/Khushboo/selenium-project"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.017">
  <properties />
            <testcase name="getLogin" time="0.017" classname="org.example.testcases.LoginTestCase">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.example.testcases.LoginTestCase.getLogin(LoginTestCase.java:20)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
  org.testng.TestRunner.privateRun(TestRunner.java:829)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
  org.testng.SuiteRunner.run(SuiteRunner.java:330)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
  org.testng.TestNG.runSuites(TestNG.java:1099)
  org.testng.TestNG.run(TestNG.java:1067)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
